boa_engine/src/vm/opcode/mod.rs

```diff
@@ -121,16 +124,86 @@ where
     unsafe { read_unchecked(bytes, offset) }
 }
 
+/// Represents a varying operand kind.
+#[derive(Default, Debug, Clone, Copy)]
+pub(crate) enum VaryingOperandKind {
+    #[default]
+    Short,
+    Half,
+    Wide,
```

Comment:
I don't think these names are descriptive of what they mean. `Short` varies per architecture, `Half` sounds like the operand will be half size the usual operand (which isn't true), and `Wide` doesn't really tell anything about the size of the operand.

Some alternatives:
- `U8`, `U16`, `U32` (preferred if still readable, but I think `VaryingOperand::u8(val)` is a bit weird)
- `Byte`, `BiByte`, `QuadByte` (preferred overall)
- `Byte`, `DoubleByte`, `QuadrupleByte` (a bit longer but more descriptive)

Reply:
The opcode is postfixed when we dump the bytecode to diffirentiate between varying kinds.

Example:

Current:
```js
000123    0509      none      PushLiteral                           2
000456    0511      none      PushLiteral.Half                      500
000789    0509      none      PushLiteral.Wide                      100000
```

Suggestion 1:
```js
000123    0509      none      PushLiteral                           2
000456    0511      none      PushLiteral.U16                       500
000789    0509      none      PushLiteral.U32                       100000
```

Suggestion 2:
```js
000123    0509      none      PushLiteral                           2
000456    0511      none      PushLiteral.BiByte                    500
000789    0509      none      PushLiteral.QuadByte                  100000
```

Suggestion 3:
```js
000123    0509      none      PushLiteral                           2
000456    0511      none      PushLiteral.DoubleByte                500
000789    0509      none      PushLiteral.QuadrupleByte             100000
```

I would prefer to use a short description `<= 5` letters to not clutter the dump too much. So I think the first suggestion is the best

Reply:
Yep, let's go with that